mmdet/models/utils/transformer.py

```diff
@@ -1052,6 +1054,7 @@ def __init__(self,
                  feat_channels=64,
                  out_channels=None,
                  input_feat_shape=7,
```

Comment:
with_proj=False

Reply:
```DynamicConv``` is used both in SparseRCNN and QueryInst. SparseRCNN use ```DynamicConv``` in ```DIIHead``` with the projection linear layer. To maintain the stability of this code, we follow the minimal modification principle and set the default setting of ```with_proj``` to ```False``` and only set it to ```True``` for ```DynamicMaskHead```.

Reply:
```DynamicConv``` is used both in SparseRCNN and QueryInst. SparseRCNN use ```DynamicConv``` in ```DIIHead``` with the projection linear layer. To maintain the stability of this code, we follow the minimal modification principle and set the default setting of ```with_proj``` to ```False``` and only set it to ```True``` for ```DynamicMaskHead```.

Reply:
So the default value should be false.

Reply:
> So the default value should be false.

I made a mistake in my statement above. SparseRCNN set the ```with_proj``` to ```True```.